website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,55 @@
+## Usage
+
+### When to use
+
+- To reveal content as needed while keeping the UI organized and visually manageable.
+- To hide long pieces of content with a similar structure.
+
+### When not to use
+
+- To hide basic non-critical information to users. Instead, consider using a [Reveal](/components/reveal).
+- To allow users switch between different content within the same context. Instead, consider using [Tabs](/components/tabs).
+
+## Toggle content
```

Comment:
Since "Toggle" is a bit of ambiguous term in this context, it might be useful to include a conceptual description of what type of content _should_ go in the toggle. Broadly off the top of my head I can think of a couple of things:

- Title content, or a title that labels the content in the content area
- Metadata type content (like a subtitle, "Consul cluster (AWS us-east-1)

I think it's important to capture here that while this acts functionally as the "toggle" for the rest of the content, it's intent is to label/title the content, even if the different types of content it accepts are variable.

Reply:
Thanks for the suggestions. I've added more details and better examples to the section.